<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="toolButtonStyle">
   <enum>Qt::ToolButtonStyle::ToolButtonIconOnly</enum>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_bmp">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>PRbpm:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_spo2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>SpO2:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_temperature">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Temperature: </string>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>711</width>
      <height>421</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_heart">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>60</y>
      <width>62</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
